pom.xml

```diff
@@ -2682,13 +2684,29 @@
       </properties>
     </profile>
     <profile>
-      <id>java9-or-later</id>
+      <id>java11-20</id>
       <activation>
-        <jdk>[9,]</jdk>
+        <jdk>[11,20]</jdk>
       </activation>
       <properties>
+        <aspectj.version>${aspectj.java11.version}</aspectj.version>
         <!-- supported since Java 9 -->
         <maven.compiler.release>${javac.version}</maven.compiler.release>
```

Comment:
nit: We could keep the original profile for setting `maven.compiler.release` (which will be the same for all JDK 9+ versions), and add the two new profiles (`[11,20]`, `[21,]`) for setting `aspectj.version`.